client/personio-timelogger.py

```diff
@@ -1,161 +1,146 @@
 #!/usr/bin/env python3
 
-import sys
-import os
 import json
+import os
 import re
-import urllib.request as request
-import urllib.parse as parse
-import http.cookiejar
-import pytz
-
-from time import sleep
+import sys
+import uuid
+from datetime import datetime, timedelta
 from random import randint
-from datetime import timedelta
-from datetime import datetime
 
-from config_sample import *
+import requests
+
 try:
-	from config import *
+    from config_sample import (
+        ATTENDANCE_URL,
+        BREAK_HOUR,
+        BREAK_TIME_MINUTES,
+        EMAIL,
+        LOGIN_URL,
+        PASSWORD,
+        PROFILE_ID,
+        SLACK_BOT_URL,
+        SLACK_MESSAGE,
+        SLACK_SECRET,
+        STARTING_HOUR,
+        WORKING_HOURS,
+    )
+
```

Comment:
In the old version, we used the `config_sample.py` as an example and the user will create a `config.py` file out of it. Take also into account that the `config.py` file is the one added to the gitignore.
A nice thing to have would be to overwrite information from the `config.py` file to the sample file.